pkgs/development/python-modules/azure-mgmt-appcontainers/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, azure-common
+, azure-mgmt-core
+, msrest
+, msrestazure
+}:
+
+buildPythonPackage rec {
+  version = "2.0.0";
+  pname = "azure-mgmt-appcontainers";
+  disabled = isPy27;
```

```suggestion
  disabled = pythonOlder "3.7";
```
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appcontainers/azure-mgmt-appcontainers/setup.py#L54